### PR TITLE
Use npm instead of yarn for everything

### PR DESCRIPTION
**Pull request by jameshowardwang**
Also upgrade merge to 1.2.1

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 tmp
 log
-.git
\ No newline at end of file
+.git
+node_modules
```
Comment (tfrcarvalho) [inline .dockerignore]: was this slowing us down before?

File: Dockerfile [hunk 2]
```diff
@@ -33,14 +29,16 @@ WORKDIR /app
...
+# Do the same for node packages, allowing them to be cached
 RUN npm update -g
+COPY package.json package-lock.json ./
+RUN npm install
 
 # Copy the main application.
 COPY . ./
 
-RUN yarn install
-RUN npm rebuild node-sass
```
Comment (tfrcarvalho) [inline Dockerfile]: So we do not need rebuild were neither, right?

Comment (jameshowardwang) [reply Dockerfile]: yeah the issue was that locally built node-sass was overwriting the one built in the docker container. adding node_modules to .dockerignore is meant to fix this.

File: .dockerignore (hunk 1 above)
Comment (jameshowardwang) [reply .dockerignore]: not slowing us down, but node_sass built locally in darwin was overwriting the one built in the docker image

**Pull request merged**